src/api/java/com/direwolf20/buildinggadgets/api/template/transaction/ITransactionExecutionContext.java

```diff
@@ -0,0 +1,9 @@
+package com.direwolf20.buildinggadgets.api.template.transaction;
+
+/**
+ * Represents some context information during execution of an {@link ITemplateTransaction}
+ */
+public interface ITransactionExecutionContext {
+    //Todo add getter for Template Boundingbox
```

Comment:
Do or remove 👍

Reply:
I can't add it without Region being present, and I'll forget if there isn't a todo to search for...

Reply:
Then space the comment :P